scripts/rick.js

```diff
@@ -4,100 +4,104 @@ const defaultSize = 40;
 const maxSize = 200;
 const intervalTimeout = 10;
 
-
 let pointerX = 0;
 let pointerY = 0;
 
-
 let rickXs = [];
 let rickYs = [];
 
-
 let increasedSizes = [];
 
+let rickCount = 0;
 
 const rickRoll = () => {
-    const image = document.createElement("img");
-    image.src = "./rick.jpg";   
-    image.className = "rick";
-    document.body.appendChild(image);
+  const image = document.createElement("img");
+  image.src = "./rick.jpg";
+  image.className = "rick";
+  increaseRickCounter();
+  document.body.appendChild(image);
 };
-       
+
 $(document).ready(() => {
   $("#click-me").on("click", rickRoll);
-    
-    document.onmousemove = function (event) { 
-        pointerX = event.pageX;
-        pointerY = event.pageY;
-    };
-    
+
+  document.onmousemove = function (event) {
+    pointerX = event.pageX;
+    pointerY = event.pageY;
+  };
+
   setInterval(() => {
-      $(".rick").each((i,image) => {
-          if (isRickAtPointer(i)) {
-              growRick(i, image);
-              return;
-          }
-
-          moveRick(i,image);
-          resetRickSize(i,image);
-      });
-  }, 
-  intervalTimeout);
+    $(".rick").each((i, image) => {
+      if (isRickAtPointer(i)) {
+        growRick(i, image);
+        return;
+      }
+
+      moveRick(i, image);
+      resetRickSize(i, image);
+    });
+  }, intervalTimeout);
 });
 
-const getDistance =(x1,y1,x2,y2) =>{
-    let y = x2 - x1;
-    let x = y2- y1;
+const getDistance = (x1, y1, x2, y2) => {
+  let y = x2 - x1;
+  let x = y2 - y1;
 
-    return Math.sqrt(x*x + y*y);
+  return Math.sqrt(x * x + y * y);
 };
 
-const getRickInfo =(i) => {
-    return { rickX: rickXs[i] ?? 0, rickY: rickYs[i] ?? 0, increasedSize: increasedSizes[i] ?? 0, };
+const getRickInfo = (i) => {
+  return {
+    rickX: rickXs[i] ?? 0,
+    rickY: rickYs[i] ?? 0,
+    increasedSize: increasedSizes[i] ?? 0,
+  };
 };
 
+const isRickAtPointer = (i) => {
+  const { rickX, rickY } = getRickInfo(i);
 
-const isRickAtPointer =(i) =>{
-    const {rickX,rickY} = getRickInfo(i);
-
-
-    const distanceFromPointer = getDistance(rickX,rickY,pointerX,pointerY);
-    return distanceFromPointer < distanceFromPointerToStop;
+  const distanceFromPointer = getDistance(rickX, rickY, pointerX, pointerY);
+  return distanceFromPointer < distanceFromPointerToStop;
 };
 
+const increaseRickCounter = () => {
+  rickCount++;
+  $("#rick-counter").text(rickCount.toString(2));
+};
```

Comment:
Took me a while to find this 💎
Would you mind splitting out the formatting changes? That would help me a lot to see what's actually new code in this pull request 🙂